src/lib.rs

```diff
@@ -259,6 +261,29 @@ mod contract {
         result.map(|_f| Vec::new()).sdk_process();
     }
 
+    #[no_mangle]
+    pub extern "C" fn register_relayer() {
+        let relayer_address = sdk::read_input();
```

Comment:
We need to check this is exactly 20 bytes long

Reply:
Good catch. Also notice that relayer can call this function any time and change it.